developer_guide/installation/system_requirements/system_requirements.rst.inc

```diff
@@ -4,11 +4,11 @@
 Akeneo PIM only works on Linux servers. The application’s behaviour has been tested on several operating systems. We cannot guarantee the behaviour of the application on servers different from:
 
 +------------------------+-------------------+
-| Debian (Linux)         | ≥ 7 (64 bits)     |
+| Debian (Linux)         | ≥ 8 (64 bits)     |
 +------------------------+-------------------+
-| Ubuntu (Linux)         | ≥ 13.04 (64 bits) |
+| Ubuntu (Linux)         | ≥ 14.04 (64 bits) |
```

Comment:
Don't know what I have to do about the ubuntu and PHP version as Ubuntu 14.04 LTS provides 5.5 by default and Ubuntu 16.04 LTS provides PHP 7 by default.